src/AlertsAggregator.py

```diff
@@ -0,0 +1,222 @@
+                alert_distribution = filtered_df['hours'].value_counts().reindex(hour_lst, fill_value=0)
+                alert_distribution = alert_distribution.to_dict()
+                distribution_lst = []
+                for i in hour_lst:
+                    alert_count_obj = AlertCountPerDay(hour=i, count=alert_distribution[i])
+                    distribution_lst.append(alert_count_obj)
+                return distribution_lst
+            elif is_settlement_exist:
+                return {"message" : f"There were no alarms in the settlement: {settlement}"}   
+            else:
+                raise ValueError("You selected a Settlement that does not exist.")
+        
+        except requests.exceptions.RequestException as req_err:
+            raise requests.exceptions.RequestException(f"Error occurred: {req_err}")
+        
+        except WrongSettlementException as settl_exc:
+            raise WrongSettlementException(settl_exc)
+        
+        except ValueError as val_err:
+            raise ValueError(val_err)   
+        
+        
+
+    # This function receives df and settlement name then count the total amount of alerts in this settlement
+    def alerts_count(self, settlement: str):
+        try:
+            user_settlement_lst= self.create_user_settl_lst(settlement)
+
+            is_settlement_exist = self.does_the_settlement_exist(settlement)
+            print(is_settlement_exist)
+            if user_settlement_lst != []:
+                # Filtered the df by the given settlement lst
+                filtered_df = self.df[self.df['data'].isin(user_settlement_lst)]
+                alert_amount = filtered_df.shape[0]
+                return {"message" : f"The total amount of alerts in {settlement} is: {alert_amount}"}
+            elif is_settlement_exist:
+                return {"message" : f"There were no alarms in the settlement: {settlement}"}
+            else:
+                raise ValueError("You selected a Settlement that does not exist.")
+        
+        except requests.exceptions.RequestException as req_err:
+            raise requests.exceptions.RequestException(f"Error occurred: {req_err}")
+        
+        except WrongSettlementException as settl_exc:
+            raise WrongSettlementException(settl_exc)
+
+        except ValueError as val_err:
+            raise ValueError(val_err)
+
+        
+    # This function receives df and count the total amount of alerts in the requested date range
+    def total_alerts_count(self):
+        total_alerts = self.df.shape[0]
+        return {"message" : f"The total amount of alerts in our country is: {total_alerts}"}
+    
+    def create_15_minute_interval_time_lst(self, start_time : time, end_time :time):
+        time_list = []
+        current_time = time(start_time.hour, 0)
+        print(current_time)
```

Comment:
use logging library.
see other comment i reffered to use logging instead of prints